src/main/java/sirius/db/mixing/properties/LocalDateProperty.java

```diff
@@ -64,16 +65,13 @@ protected LocalDateProperty(EntityDescriptor descriptor, AccessPath accessPath,
 
     @Override
     public Object transformValue(Value value) {
-        return NLS.parseUserString(LocalDate.class, value.asString());
-    }
-
-    @Override
-    protected Object transformValueFromImport(Value value) {
         if (value.is(LocalDate.class)) {
             return value.get();
         }
-
-        return transformValue(value);
+        if (value.is(LocalDateTime.class)) {
+            return value.get(LocalDateTime.class, LocalDateTime.now()).toLocalDate();
```

Comment:
```suggestion
            return value.get(LocalDateTime.class, null).toLocalDate();
```
-> Der Default sollte eh nie gebraucht werden. Wenn er je gebraucht würde, dann sollts IMHO lieber knallen